### PR TITLE
org: jupyter prerequisites

### PR DESCRIPTION
**Pull request by benneti**
Add some more instructions to use +jupyter for jupyter babel blocks.

### CODE REVIEW

File: modules/lang/org/README.org [hunk 1]
```diff
@@ -173,6 +173,8 @@ esoteric features:
 + To execute babel code blocks, you need whatever dependencies those languages
   need. It is recommended you enable the associated =:lang= module and ensure
   its dependencies are met, e.g. install the =ruby= executable for ruby support.
+  To use ~jupyter kernels~ you need the =+jupyter= flag, the associatet kernel as
```
Comment (hlissner) [inline modules/lang/org/README.org:176]: `s/associatet/associated`

File: modules/lang/org/README.org [hunk 2]
```diff
@@ -185,13 +187,17 @@ brew install gnuplot
 #+BEGIN_SRC sh
 pacman -S texlive-core texlive-bin texlive-science
 pacman -S gnuplot
+# +jupyter needs jupyter
+pacman -S jupyter
```
Comment (hlissner) [inline modules/lang/org/README.org:191]: Make it one line:

```sh
pacman -S jupyter  # required by +jupyter
```

File: modules/lang/org/README.org [hunk 3]
```diff
@@ -185,13 +187,17 @@ brew install gnuplot
...
 #+END_SRC
 
 ** NixOS
 #+BEGIN_SRC nix
 environment.systemPackages = with pkgs; [
   # any less than medium isn't guaranteed to work
   texlive.combined.scheme-medium
+  # +jupyter needs jupyter
```
Comment (hlissner) [inline modules/lang/org/README.org:199]: Reword: `# required by +jupyter`

**Review by hlissner — CHANGES_REQUESTED** [review]

**Comment by benneti** [comment]
thanks for the input, changed as requested

**Review by hlissner — APPROVED** [review]

**Comment by hlissner** [comment]
Sorry for the long wait. Thanks for your help!

**Pull request merged**